File: bindings/ruby/samples/dmm_test.rb
#!/usr/bin/env ruby

$:.unshift File.join(File.dirname(__FILE__), ".")
require 'tbi'

dmm = Tbi::Dmm.new
dmm.open()

puts sprintf("%.3f [V]", dmm.getVoltage())
puts sprintf("%.3f [mA]", 1000.0 * dmm.getCurrent())

File: bindings/python/samples/dmm_test.py
from toolbit import Dmm

dmm = Dmm()
dmm.open()
print(str('%03.3f' % dmm.getVoltage()) + " [V]")
print(str('%03.3f' % (1000.0 * dmm.getCurrent())) + " [mA]")
